pkg/controller/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -315,3 +325,43 @@ func (r *ReconcileOCSInitialization) Reconcile(request reconcile.Request) (recon
 
 	return reconcile.Result{}, err
 }
+
+// returns a ConfigMap with default settings for rook-ceph operator
+func newRookCephOperatorConfig(namespace string) *corev1.ConfigMap {
+	var defaultCSIToleration string = `
```

Comment:
Shouldn't this be generated from `NodeTolerationKey`?

Reply:
fixed.